Library/Formula/llvm.rb

```diff
@@ -94,7 +94,7 @@ class Llvm < Formula
   deprecated_option "rtti" => "with-rtti"
   deprecated_option "disable-assertions" => "without-assertions"
 
-  if MacOS.version <= :snow_leopard
+  if MacOS.version <= :snow_leopard || build.with?("python")
```

Comment:
So it turns out that using `depends_on :python` _doesn't_ bring Python in as a dependency but `depends_on "python"` _does_. Is this a bug in Homebrew or in the Formula?